app/src/main/java/com/friendinneed/ua/friendinneed/MainActivity.java

```diff
@@ -426,7 +427,9 @@ protected void onPostExecute(Void aVoid) {
             @Override
             protected Void doInBackground(Void... params) {
                 try {
-                    response = post("http://192.168.43.184:8086", new Gson().toJson(sample));
+                    //response = post("http://192.168.43.184:8086", new Gson().toJson(sample));
+                    response = postDataQueueSamples("http://friendinneedserver6099.cloudapp.net:5000", queue.dump());
```

Comment:
seems like You added '/' api call to obsolete code. this asynctask pretend to ask server if fall data "looks like" person fall. for now we need to retrain our neural network and just collect data. so we should change logic here. So here we go: 1 we check all incoming acceleration data for threshold 2 if we reach threshold we launch TickerDialog (or LabelingDialog) 3 user choose yes/no - we set label to 1.0 or 0.1 respectively in data sample 4 we send labeled data sample to '/' api
I will add new api call to be used here to be able to decide if user falls to the ground later